### ISSUE TITLE
기본 함수 완성 + 제안사항

### ISSUE THREAD

**Issue by jeeseongmin** [jeeseongmin/oss_miniproject#14]
CRUD와 file save, upload 함수는 모두 테스트까지 완료했습니다.
그리고 일단 거짓 데이터로써, [oss_score.txt] 를 넣어놨는데, 아무 학생이름과 나머지 점수는 0점으로 설정해놓았습니다.

**제안 사항 :** 
(1) sum, average 함수
(2) 총 점수를 토대로 등수를 보여주는 함수
(3) 2번 항목과 비슷한데, grade를 매겨주는 함수

하다보니 있으면 좋을 것 같은 함수를 나열해보았습니다.
시간 날 때 작업하면 될 것 같습니다.
+ 1차 테스트는 제가 했지만, 학우님도 한번 해주셨으면 좋겠습니다. (CRUD, file save, upload)

**Comment by ykeab94** [comment]
네 저도 위에 제안된 함수를 만들어보고,  CRUD와 file save, upload 함수를 테스트 해보겠습니다!
